src/models/block.rs

```diff
@@ -91,7 +91,8 @@ pub fn rpc_to_primitive_header(
         state_root: header.state_root,
         timestamp: header.timestamp.try_into().map_err(|_| EthereumDataFormatError::PrimitiveError)?,
         transactions_root: header.transactions_root,
-        withdrawals_root: header.withdrawals_root,
+        // Withdrawals are not allowed so we push a None value
+        withdrawals_root: Default::default(),
```

Comment:
Here it is done in a safe way because we already check beforehand that the root withdrawal corresponds to the keccak of the root empty. If we put a value here, this deprives us of the possibility of doing random tests because the root withdrawal is an rlp encoding of several values as explained here https://eips.ethereum.org/EIPS/eip-4895.

So if we want the rlp to pass with a root withdrawal not `None`, then we must satisfy conditions on `gas_limit`, `gas_used` or `base_fee_per_gas` for example. Which is useless because ultimately withdrawals are not supported by the RPC.